mdv.js

```diff
@@ -34,45 +34,43 @@ const argv = require('yargs')
 let exitCode = 0;
 const options = argv;
 
-
 for (let a of argv._) {
-    glob(a, options, function (er, files) {
-        for (let file of files) {
-            const s = fs.readFileSync(a,'utf8');
-            options.source = file;
-            const result = validator.validate(s,options);
+    const files = glob.sync(a, { debug: !!options.debug });
+    for (let file of files) {
+        const s = fs.readFileSync(a,'utf8');
```

Comment:
I think this `readFileSync` should be of `file` not `a`.

Reply:
Good catch! I had it on my first iteration and lost it for some reason.